modules/exploits/linux/http/freepbx_hotelwakeup_rce.rb

```diff
@@ -0,0 +1,120 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit4 < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'            => 'FreePBX < 13.0.188.1 Remote root exploit',
+                      'Description'     => '
+                        This module exploits an unauthenticated remote command execution in FreePBX module Hotelwakeup
```

Comment:
This is where you can include that version number, and the fact that it gets root